src/plays/quizeo/src/components/menu-item/menuItem.component.js

```diff
@@ -12,8 +11,8 @@ const MenuItem = ({title, imageUrl, linkUrl}) => {
     }
 
     const handleLocation = () => {
-        if(window.location.pathname === '/movie'){  
-            <MoviePage categoryId={linkUrl}/>
+        if(window.location.pathname === '/plays/quizeo/movie'){  
+            return <MoviePage categoryId={linkUrl}/>
         }
```

Comment:
The component didn't render on the new page. I tried but didn't get any solution. Can you please have a look?

Reply:
ok... 

What we discussed in to get rid of the routing within the play. So my thought was, it is single page app when it goes into the quiz play.. after that you change the component based on the actions but it doesn't change the route. Let's keep it simple in a way that it demonstrate the idea of redux but don;t have to touch routing.

I know it will be a bit different implementation than your current project but the changes will suite well to ReactPlay. Let me know what do you think?